mifiStatus/mifiStatus.js

```diff
@@ -1,16 +1,15 @@
 const AWS = require('aws-sdk');
-const ExpenseType = require('./models/expenseType');
-const DatabaseModify = require('./js/databaseModify');
-const ExpenseRoutes = require('./routes/expenseRoutes');
-const fs = require('fs');
 const FormData = require('form-data');
-const dateUtils = require('./js/dateUtils');
-const { generateUUID } = require('./js/utils');
+const fs = require('fs');
+const ExpenseType = require('expenseType'); // from shared layer
+const DatabaseModify = require('databaseModify'); // from shared layer
+const ExpenseRoutes = require('expenseRoutes'); // from shared layer
```

Comment:
Shouldn't these references use the same ternary syntax, for consistency and running locally?